src/test/java/org/opensearch/geospatial/plugin/GeospatialPluginTests.java

```diff
@@ -25,4 +37,26 @@ public void testFeatureProcessorIsAdded() {
         assertTrue(processors.containsKey(FeatureProcessor.TYPE));
         assertTrue(processors.get(FeatureProcessor.TYPE) instanceof FeatureProcessor.Factory);
     }
+
```

Comment:
Im wondering if this test case is really necessary. If its for test coverage, could we just ignore the plugin class?

Reply:
FeatureProcessor is added to IngestPlugin only through getProcessor. I don't know how to test whether the processor is added or not using unit test. Thats the reason i added this test.

Reply:
Yeah, the below methods are added to make sure that all relevant classes are added to the list (not for logic ). I am not sure excluding is the right way to go. Is there any other way to assert that all relevant class(RestHandlers/TransportAction) is added to plugin?

Reply:
In terms of relevant classes added to the plugin, I think these will be tested by other test cases - if the classes are missing the other tests will fail. 

This is not a blocker. Ill approved. Im just not sure what it will catch that other tests wont.